curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657ef2a86d4e545cec9a85fb.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 657ef2a86d4e545cec9a85fb
+title: Step 4
+challengeType: 20
+dashedName: step-4
+---
+
+# --description--
+
+Uppercase characters in camel case or pascal case indicate the start of new words. Inside the loop body, use an `if` statement in conjunction with the `.isupper()` string method to check for uppercase characters.
+
+# --hints--
+
+You should write a new `if` statement. Don't forget the colon at the end.
+
+```js
+assert.match(code, /if char\.isupper\(\):/i)
+```
+
```

Comment:
I'm not able to build the client. Here, a hint is required between the two code blocks. That should solve the issue.